hooks/before_prepare/icon.js

```diff
@@ -144,6 +148,7 @@ Promise.resolve()
 		<% if(errorHandlingStrategy === 'warn') { %>
 		console.warn(err.message);
 		<% } else if(errorHandlingStrategy === 'throw') { %>
+		console.error(err.message);
```

Comment:
Throwing doesn't abort the operation (I think it's squelched somewhere). So at least you see an error message if you read the logs. Maybe this line shouldn't be in this pull request and I should file a ticket or create another pull request.